tests/kbd_test.py

```diff
@@ -487,6 +489,14 @@ def test_bcache_get_set_mode(self):
         wipe_all(self.loop_dev, self.loop_dev2)
 
 class KbdTestBcacheStatusTest(KbdBcacheTestCase):
+
+    def _get_size(self, bcache_name):
+        cache_dir = '/sys/block/%s/bcache/cache' % bcache_name
+
+        # sum sizes from all caches
+        caches = ['%s/%s' % (cache_dir, d) for d in os.listdir(cache_dir) if re.match('cache[0-9]*$', d)]
```

Comment:
I think it's better to use ``glob.glob()`` here.

Reply:
I took this from udisks tests -- https://github.com/storaged-project/udisks/blob/master/src/tests/dbus-tests/test_bcache.py#L56 -- glob would be better, but I think this is ok for tests.